timdex_dataset_api/metadata.py

```diff
@@ -249,18 +264,30 @@ def setup_duckdb_context(self) -> DuckDBPyConnection:
             3. Create additional metadata views as needed.
 
         The resulting, in-memory DuckDB connection is used for all metadata queries.
+
+        If a static database file is not found, a configured DuckDB connection is still
+        returned.
         """
+        start_time = time.perf_counter()
+
         conn = duckdb.connect()
         self.configure_duckdb_connection(conn)
 
-        if self.database_exists():
-            self._attach_database_file(conn)
-        else:
+        if not self.database_exists():
             logger.warning(
                 f"Static metadata database not found @ '{self.metadata_database_path}'. "
                 "Please recreate via TIMDEXDatasetMetadata.recreate_database_file()."
             )
+            return conn
```

Comment:
Realized first half of [this comment ](https://github.com/MITLibraries/timdex-dataset-api/pull/157#discussion_r2255102350) was lost in the mix as the thread focused on the "static" part of the method name. Re-asking question: 
> Is TIMDEXDatasetMetadata.recreate_static_database_file() also used when the database is created for the first time? 🤔 If so, I feel the name is a bit misleading.